### PR TITLE
Loading resources properly

### PR DESCRIPTION
**Pull request by bimusiek**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 bower_components/
+.idea
```
Comment (krzkaczor) [inline .gitignore]: why? i mean IMO we should remove it completely from git repo but ignoring it only won't solve it

File: dist/angular-globalize-wrapper.js [hunk 2]
```diff
@@ -68,18 +68,29 @@ glModule.provider('globalizeWrapper', function () {
...
+                allResources = allResources.concat(mainResources.map(function(mr) {
+                  return '/'+locale+mr;
+                }));
+              }
+              loadResources(
+                  cldrBasePath + '/main/',
+                  allResources,
+                  function (results) {
+                      mainData = [];
+                      for (var i = 0; i < results.length; i++)
```
Comment (ghost) [inline dist/angular-globalize-wrapper.js:86]: `{ ... }` ;)

File: js/globalize.js [hunk 3]
```diff
@@ -65,18 +65,29 @@ glModule.provider('globalizeWrapper', function () {
...
+                allResources = allResources.concat(mainResources.map(function(mr) {
+                  return '/'+locale+mr;
+                }));
+              }
+              loadResources(
+                  cldrBasePath + '/main/',
+                  allResources,
+                  function (results) {
+                      mainData = [];
+                      for (var i = 0; i < results.length; i++)
```
Comment (ghost) [inline js/globalize.js]: Same here ;)

Comment (ghost) [inline js/globalize.js]: Actually, map won't work here? :D

Comment (bimusiek) [inline js/globalize.js]: I dont want to change old code that is not mine :) I am not refactoring but fixing.

**Pull request merged**